spec/openapi.yaml

```diff
@@ -1138,6 +1159,56 @@ components:
                 description: Path to package
                 example: "jo/packages"
 
+    Job:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 420
+        name:
+          type: string
+          description: Short Filename
+          example: "SPICE~1.gco"
+        path:
+          type: string
+          example: "/local"
+        display_name:
+          type: string
+          description: Long Filename
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        display_path:
+          type: string
+          example: "/PrusaLink gcodes/"
```

Comment:
I'm getting more and more confused. The example would suggest this is only the directory it lives in, not complete path?